docs/private-dns/connect-devices/routers/mikrotik.md

```diff
@@ -45,16 +45,20 @@ MikroTik routers use the open source RouterOS operating system, which provides r
 
 ## Use your router admin panel
```

Comment:
Rename to "My router does not support DNS-over-HTTPS"

Reply:
let's leave this header as is — it's one of the ways to set up Private AdGuard DNS on Mikrotic. added a short explanation above to make it more clear for users